lib/Dialect/TritonGPU/Transforms/ReduceDataDuplication.cpp

```diff
@@ -76,8 +76,8 @@ class TritonGPUReduceDataDuplicationPass
               srcType.getElementType()));
       auto tmp = builder.create<triton::gpu::LocalAllocOp>(
           cvtOp.getLoc(), tmpType, cvtOp.getSrc());
-      auto newConvert = builder.create<triton::gpu::LocalLoadOp>(cvtOp.getLoc(),
-                                                                 dstType, tmp);
+      auto newConvert = builder.create<triton::gpu::LocalLoadOp>(
+          cvtOp.getLoc(), dstType, tmp, nullptr);
```

Comment:
nit: can you add a new constructor for LocalLoadOp so that we don't have to pass nullptr everywhere?